content/5_Plat/2DRQ.mdx

```diff
@@ -322,93 +344,89 @@ And you might use it like so:
 
 <LanguageSection>
 <CPPSection>
+
 ```cpp
-#include <algorithm>
-#include <cassert>
-#include <iostream>
-#include <vector>
+#include <bits/stdc++.h>
```

Comment:
:(

Reply:
do u want me to change it to have each include separately

Reply:
i'd like that